pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1095,19 +1117,14 @@ func newAWSDisk(aws *AWSCloud, name string) (*awsDisk, error) {
 	if strings.Contains(awsID, "/") || !strings.HasPrefix(awsID, "vol-") {
 		return nil, fmt.Errorf("Invalid format for AWS volume (%s)", name)
 	}
-	az := url.Host
-	// TODO: Better validation?
-	// TODO: Default to our AZ?  Look it up?
-	// TODO: Should this be a region or an AZ?
-	if az == "" {
-		return nil, fmt.Errorf("Invalid format for AWS volume (%s)", name)
-	}
-	disk := &awsDisk{ec2: aws.ec2, name: name, awsID: awsID, az: az}
+	// az := url.Host
```

Comment:
Yes, for now.  I'll change it to a comment that explains what the original motivation for the zone being there was, even if it turns out that we don't need it (because we're using labels instead)...